doc/mlsys_paper/paper.tex

```diff
@@ -301,48 +305,50 @@ \section{Types of objective functions}
 \vspace*{-0.3em}
 
 In this framework, if a user wants to optimize a differentiable objective
-function, they need only provide implementations of $f(x)$ and $f'(x)$, and
+function, they only need to provide implementations of $f(x)$ and $f'(x)$, and
```

Comment:
I did that to reclaim a little space, but if it's not necessary "only need to" reads better.

Reply:
"they need only" sounds weird - like something out of Shakespeare :)